test/e2e/nnce_conditions_test.go

```diff
@@ -105,6 +105,9 @@ var _ = Describe("EnactmentCondition", func() {
 		})
 
 		AfterEach(func() {
+			By("Remove the bridge")
+			updateDesiredState(linuxBrAbsent(bridge1))
+			waitForAvailableTestPolicy()
```

Comment:
I'm not at all sure, but based on the /name/ of resetDesiredStateForNodes(), I would expect that cleaning the nodes should happen inside it.

Reply:
Resed desired state is removing the policy object. But removal of NNCP does not remove the interfaces it configured. Hence, we first need to remove what we created and only then remove the policy object.

Reply:
isn't this order raceful? After the bridge is removed, the policy controller may wake up and re-create it. Or am I plain wrong?

Reply:
We changed the original policy with the `absent` desired state. So there should be no race.